cmd/telem_view/telem_view.go

```diff
@@ -18,6 +20,10 @@ import (
 const valueColWidth = 12
 
 var shmDir = flag.String("shm", "/dev/shm", "directory to use for shared memory")
+var fpsLimit = flag.Int("fps", 0, "max UI frames per second (0 = unlimited)")
+
+var updateCounter uint64
+var pendingUpdate int32
```

Comment:
Could these be their atomic equivalents? They're shared across threads and all operations to them are already atomic.